sdk/storage/Azure.Storage.Common/src/Shared/ClientsideEncryption/Models/EncryptionAgent.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage.Cryptography.Models
+{
+    /// <summary>
+    /// Represents the encryption agent stored on the service.
+    /// </summary>
+    internal class EncryptionAgent
+    {
+        /// <summary>
+        /// The protocol version used for encryption.
+        /// </summary>
+        public ClientSideEncryptionVersion Protocol { get; set; }
```

Comment:
Nit - I'd suggest naming this `EncryptionVersion` instead of `Protocol` to match the name in Options.